package.json

```diff
@@ -3,7 +3,7 @@
   "version": "1.0.0",
   "private": true,
   "dependencies": {
-    "@bandwidth/webrtc-browser": "^0.6.0",
+    "@bandwidth/webrtc-browser": "beta",
```

Comment:
Obviously temporary, but will hopefully let us iterate on the SDK without having to touch this repo unless needed.